mods/cnc/maps/gdi04c/gdi04c.lua

```diff
@@ -41,7 +40,6 @@ end
 
 TownAttackAction = function(actor)
 	Trigger.OnIdle(actor, TownAttackersIdleAction)
-	FollowWaypoints(actor, TownAttackWpts)
```

Comment:
Why remove this?

Reply:
`waypoint1` and `waypoint2` are local `nil` values. This is dead code.